### PR TITLE
Add Benchmark 2.0

### PR DESCRIPTION
**Pull request by Naarcha-AWS**
### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _benchmark/user-guide/installing-benchmark.md [hunk 1]
```diff
@@ -199,7 +199,7 @@ After running OpenSearch Benchmark for the first time, you can search through al
 * `benchmark.ini`: Contains any adjustable configurations for tests. For information about how to configure OpenSearch Benchmark, see [Configuring OpenSearch Benchmark]({{site.url}}{{site.baseurl}}/benchmark/configuring-benchmark/).
 * `data`: Contains all the data corpora and documents related to OpenSearch Benchmark's [official workloads](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/geonames).
 * `distributions`: Contains all the OpenSearch distributions downloaded from [OpenSearch.org](http://opensearch.org/) and used to provision clusters.
-* `test_executions`: Contains all the test `execution_id`s from previous runs of OpenSearch Benchmark.
+* `test_runs`: Contains all the test `execution_id`s from previous runs of OpenSearch Benchmark.
```
Comment (IanHoang) [inline _benchmark/user-guide/installing-benchmark.md:202]: `execution_id`s -> `run_id`s

File: _benchmark/reference/commands/run.md [hunk 2]
```diff
@@ -142,21 +143,21 @@ The following options help customize how OpenSearch Benchmark provisions OpenSea
...
-The following options determine which workload is used to run the test:
+The following options determine which workload is used during the run:
 
 - `--workload-repository`: Defines the repository from which OpenSearch Benchmark loads workloads. 
 - `--workload-path`: Defines the path to a downloaded or custom workload.
 - `--workload-revision`: Defines a specific revision from the workload source tree that OpenSearch Benchmark should use.
 - `--workload`: Defines the workload to use based on the workload's name. You can find a list of preloaded workloads using `opensearch-benchmark list workloads`.
 
-### Test procedures
+### Run procedures
```
Comment (IanHoang) [inline _benchmark/reference/commands/run.md:153]: Run Procedures -> Scenarios

File: _benchmark/reference/commands/run.md [hunk 3]
```diff
@@ -142,21 +143,21 @@ The following options help customize how OpenSearch Benchmark provisions OpenSea
...
 
-The following options define what test procedures the test uses and which operations are contained inside the procedure: 
+The following options define what run procedures the run uses and which operations are contained inside the procedure: 
 
-- `--test-execution-id`: Defines a unique ID for this test run.
-Defines the test procedures to use with each workload. You can find a list of test procedures that the workload supports by specifying the workload in the `info` command, for example, `opensearch-benchmark info --workload=<workload_name>`. To look up information on a specific test procedure, use the command `opensearch-benchmark info --workload=<workload_name> --test-procedure=<test-procedure>`.
-- `--include-tasks`: Defines a comma-separated list of test procedure tasks to run. By default, all tasks listed in a test procedure array are run.
-- `--exclude-tasks`: Defines a comma-separated list of test procedure tasks not to run.
+- `--test-run-id`: Defines a unique ID for this run.
```
Comment (IanHoang) [inline _benchmark/reference/commands/run.md:157]: Should we simplify this to `test-id` instead of `test-run-id`? @gkamat We haven't made this change yet so it's possible to switch to `test-id` to keep it simple...

File: _benchmark/reference/commands/run.md [hunk 4]
```diff
@@ -142,21 +143,21 @@ The following options help customize how OpenSearch Benchmark provisions OpenSea
...
 - `--workload-repository`: Defines the repository from which OpenSearch Benchmark loads workloads. 
 - `--workload-path`: Defines the path to a downloaded or custom workload.
 - `--workload-revision`: Defines a specific revision from the workload source tree that OpenSearch Benchmark should use.
 - `--workload`: Defines the workload to use based on the workload's name. You can find a list of preloaded workloads using `opensearch-benchmark list workloads`.
 
-### Test procedures
+### Run procedures
 
-The following options define what test procedures the test uses and which operations are contained inside the procedure: 
+The following options define what run procedures the run uses and which operations are contained inside the procedure:
```
Comment (IanHoang) [inline _benchmark/reference/commands/run.md:155]: run procedures --> scenarios
the run uses --> the test uses

File: _benchmark/reference/commands/run.md [hunk 5]
```diff
@@ -142,21 +143,21 @@ The following options help customize how OpenSearch Benchmark provisions OpenSea
 
 ### Workload
 
-The following options determine which workload is used to run the test:
+The following options determine which workload is used during the run:
```
Comment (IanHoang) [inline _benchmark/reference/commands/run.md:146]: I think `during the test` sounds more natural

File: _benchmark/reference/commands/run.md [hunk 6]
```diff
@@ -86,11 +87,11 @@ After the test runs, OpenSearch Benchmark responds with a summary of the benchma
 
 ## Options
 
-Use the following options to customize the `execute-test` command for your use case. Options in this section are categorized by their use case. 
+Use the following options to customize the `run` command for your use case. Options in this section are categorized by their use case. 
 
 ## General settings
 
-The following options shape how each test runs and how results appear: 
+The following options customize each run and affects how run results appear:
```
Comment (IanHoang) [inline _benchmark/reference/commands/run.md:94]: I think we can reword this to be --> ...customize each test run and affects how results appear

File: _benchmark/reference/commands/run.md [hunk 7]
```diff
@@ -1,25 +1,26 @@
...
+# run
 <!-- vale on -->
 
-Whether you're using the included [OpenSearch Benchmark workloads](https://github.com/opensearch-project/opensearch-benchmark-workloads) or a [custom workload]({{site.url}}{{site.baseurl}}/benchmark/creating-custom-workloads/), use the `execute-test` command to gather data about the performance of your OpenSearch cluster according to the selected workload. 
+Whether you're using the included [OpenSearch Benchmark workloads](https://github.com/opensearch-project/opensearch-benchmark-workloads) or a [custom workload]({{site.url}}{{site.baseurl}}/benchmark/creating-custom-workloads/), use the `run` command to gather data about the performance of your OpenSearch cluster according to the selected workload. 
 
 ## Usage
 
-The following example executes a test using the `geonames` workload in test mode: 
+The following example tests a run using the `geonames` workload in test mode:
```
Comment (IanHoang) [inline _benchmark/reference/commands/run.md:20]: Might be more clear to use: The following example performs a quick test using the `geonames` workload in test mode.

**Review by IanHoang — CHANGES_REQUESTED** [review]
Some changes requested. Also, is it possible to have this merged in only when we release 2.0.0?

File: _benchmark/reference/commands/run.md (hunk 6 above)
Comment (Naarcha-AWS) [inline _benchmark/reference/commands/run.md:94]:
```suggestion
The following options customize each test run and affects how run results appear: 
```

File: _benchmark/reference/commands/run.md (hunk 5 above)
Comment (Naarcha-AWS) [inline _benchmark/reference/commands/run.md:146]:
```suggestion
The following options determine which workload is used during the test:
```

File: _benchmark/reference/commands/run.md (hunk 2 above)
Comment (Naarcha-AWS) [inline _benchmark/reference/commands/run.md:153]:
```suggestion
### Scenarios
```

File: _benchmark/reference/commands/run.md (hunk 4 above)
Comment (Naarcha-AWS) [inline _benchmark/reference/commands/run.md:155]:
```suggestion
The following options define what scenarios the test uses and which operations are contained inside the procedure: 
```

File: _benchmark/reference/workloads/scenarios.md [hunk 8]
```diff
@@ -1,25 +1,27 @@
 ---
 layout: default
-title: test_procedures
+title: Scenarios
 parent: Workload reference
 grand_parent: OpenSearch Benchmark Reference
 nav_order: 110
+redirect_from:
+  - /benchmark/workloads/test-procedures/
```
[the remark this answers is not in the document]
Comment (Naarcha-AWS) [reply _benchmark/reference/workloads/scenarios.md:8]: That is the permalink this redirect is targeting.

File: _benchmark/user-guide/installing-benchmark.md (hunk 1 above)
Comment (Naarcha-AWS) [inline _benchmark/user-guide/installing-benchmark.md:202]:
```suggestion
* `test_runs`: Contains all the test `test_run_id`s from previous runs of OpenSearch Benchmark.
```

**Comment by Naarcha-AWS** [comment]
> Some changes requested. Also, is it possible to have this merged in only when we release 2.0.0?

We'll hold the PR until then!

**Comment by IanHoang** [comment]
Let's close this PR as we don't have a concrete date yet. OSB 2.0.0 should be thoroughly tested before we release. We'll revisit this once we have an idea of when we're going to release it.

**Comment by Naarcha-AWS** [comment]
Closing until we have a firm date for Benchmark 2.0

**Comment by Naarcha-AWS** [comment]
Since this is still TBD, going to close this and loop back around when necessary.
